usaspending_api/settings.py

```diff
@@ -845,3 +845,23 @@
     "original_loan_subsidy_cost": "original_loan_subsidy_cost",
     "business_funds_indicator": "business_funds_indicator"
 }
+
+# If caches added or renamed, edit clear_caches in usaspending_api/etl/helpers.py
+CACHES = {
```

Comment:
I'd echo this - I know in contracts and assistance we build a few lists (IE txn) - would we need to insert a "recipient_cache = caches['recipients']" for the two loaders? 

Wasn't sure if the default cache just automatically stored everything you've already pulled! I know we clear the caches before you run the rest of the code in the loader too.